README.md

```diff
@@ -1 +1,68 @@
 # fastify-queue
+An async queue for [Fastify](https://www.fastify.io/docs/latest/).
+
+## Usage
+
+```sh
+npm i @autotelic/fastify-queue
+```
+#### Example
+
+```js
+function myRoute (fastify, opts) {
+  fastify.register(fastifyQueue, {
+    initiator: async (key, value, reply) => {
+      return reply.myAsyncAction(value)
+    }
+  })
+
+  fastify.post('/example', async (request, reply) => {
+    reply.queue.add('myItem', request.body)
```

Comment:
Just so I understand, the example here is essentially a noop.
In a webhook example, the value here (`request.body`) would be, say, a function that posts to an endpoint?

Reply:
My bad, the example is a bit sparse. For a webhook example, yes you could have the value be a function or unresolved promise. But we also provide an `initiator` option (which is being used in the example) this provides a means of reducing boilerplate. So if you're always calling the same function to post your webhooks you could make that function an `initiator` and then when you want to add a webhook to your queue just:
```js
// before an item is added to the queue it's passed into the initiator and the value returned is then added to the queue
reply.queue.add('webhookOne', postArgs)

// Alternatively you could have no initiator and just pass in your async functions.
reply.queue.add('webhookOne', postFn('/endpoint', options))
```
Both are ways valid